Plugins/Flow.Launcher.Plugin.PluginsManager/PluginsManager.cs

```diff
@@ -311,7 +375,11 @@ internal async ValueTask<List<Result>> RequestInstallOrUpdate(string searchName,
 
             token.ThrowIfCancellationRequested();
 
-            var searchNameWithoutKeyword = searchName.Replace(Settings.HotKeyInstall, string.Empty).Trim();
+            var searchNameWithoutKeyword = searchName.Replace(Settings.HotKeyInstall, string.Empty, StringComparison.OrdinalIgnoreCase).Trim();
+
+            if (Uri.IsWellFormedUriString(searchNameWithoutKeyword, UriKind.Absolute) 
```

Comment:
what if the url doesn't end with zip?

Reply:
then no option to download, we support zip currently